### PR TITLE
docs: improve templates and create template config with external links

### PR DESCRIPTION
**Pull request by MuckT**
### Description

This is an update to our issue templates including a more detailed bug report template, a feature request template, and a `config.yaml` to house external links when users go to submit an issue. This is to help bin things more appropriately and prevent support requests from being submitted as tickets. Also it includes a link to how to submit a security related issue and many other CELO & cLabs links in the `config.yaml`

### Tested

The changes were tested by viewing the `.md` files in preview mode. This will not need to be tested in the release cycle as it doesn't impact application code; however, product review is needed be help to confirm stylistic wording.

### Backwards compatibility

These changes should be backwards compatible and only impact new issues.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1-bug-report.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Have a question about development? Join our Discord 👉 https://discord.gg/7tKnCbHv5j
+
+As an open source project - it may take some time for your issue to be addressed. Please be patient and we will respond as soon as we can. 🙏 -->
+
+### Current behavior
+<!-- A description including screenshots, videos, etc. -->
+
+### Desired behavior
+<!-- Remember, we are not familiar with the application you're testing, so please provide a clear description of what should happen.-->
```
Comment (tarikbellamine) [inline .github/ISSUE_TEMPLATE/1-bug-report.md:21]: I'm a little confused by the phrasing `we are not familiar with the application you're testing`. Isn't this for Valora?

File: .github/ISSUE_TEMPLATE/config.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 🙋 Get Support for your issues
+    url: https://valoraapp.com/support
+    about: This issue tracker is not for support questions. Please refer to our support page or submit an in-app support ticket.
```
Comment (tarikbellamine) [inline .github/ISSUE_TEMPLATE/config.yaml:5]: Perhaps some very brief instructions for how to report in-app support issues?

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -11,6 +11,8 @@ _Describe any minor or "drive-by" changes here._
 
 _An explanation of how the changes were tested or an explanation as to why they don't need to be._
 
+_Does this need to be tested by QA in the next release cycle? If so please give a brief explanation of how to test these changes._
```
Comment (tarikbellamine) [inline .github/pull_request_template.md:14]: Can we put this under it's own header? I'm afraid it will be missed if included within `Description`?

**Review by tarikbellamine — COMMENTED** [review]
Good stuff! Left some comments

File: .github/ISSUE_TEMPLATE/config.yaml (hunk 2 above)
Comment (MuckT) [reply .github/ISSUE_TEMPLATE/config.yaml:5]: Adjusted instructions in commit [74d6270](https://github.com/celo-org/wallet/pull/281/commits/74d627050c69ba9ab964631a6b84fbb2cfa149ec)

File: .github/pull_request_template.md (hunk 3 above)
Comment (MuckT) [reply .github/pull_request_template.md:17]: Moved to its own section in commit [a78b97b](https://github.com/celo-org/wallet/pull/281/commits/a78b97bbf03aff485624e328583c0264e3456c8c)

**Review by tarikbellamine — APPROVED** [review]
Good stuff! Thanks for doing it this

**Pull request merged**